### DISCUSSION TITLE
Hello, I have to remove the BMS JK from Victron Venus OS.

### DISCUSSION THREAD

**Discussion by 21-Jailbraik-21** [Louisvdw/dbus-serialbattery#478] [Q&A] [answered]
Hello, I have to remove the BMS JK from Victron Venus OS.
How do I do that? If I now pull off the cable, I get EIINE error message immediately.

Thanks

**Comment by Louisvdw** [comment, marked as answer]
You can follow this
https://github.com/Louisvdw/dbus-serialbattery/wiki/How-to-install#disable

**Comment by 21-Jailbraik-21** [comment]
Thanks, didn't understand that I had to reset the MPPT.
I thought I have to make settings in Venus OS.
Thanks
